GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/SpaceViewModel.cs

```diff
@@ -150,6 +150,7 @@ private void OnShowError(string message)
 
         private void OnMoveViewWest(object obj)
         {
+            GlobalData.GetInstance().Logger?.AddEntry("Move_Map", context: "West");
```

Comment:
Good call. This makes me wonder: are there other places in the app that we might anticipate issues? Might be useful to have a list of "risky" spots to go back and add logs to if something starts happening that mystifies us. No change required to this PR, though.

Reply:
There could be, and this comes up sometimes as we chat about the information we want to collect through the touch table. It's definitely something worth keeping in mind. The logs do a good job telling us what people are doing, but they also have the benefit of informing us that people are doing things we didn't anticipate (such as hitting the max level when classifying).